upgrade_versions.py

```diff
@@ -188,6 +193,21 @@ def _bump_release(self, pkg, log=None, user_name=None, user_email=None):
             assert user_email is not None
             rpm_bump_spec(pkg.specfile, log, user_name, user_email)
 
+    def _update_prerelease_tag(self):
+        if not self._prerelease:
+            self._prerelease = "%{nil}"
```

Comment:
Why do we need to set to `"%{nil}" here?` If we don't have a pre-release, no string will be replaced in the file. You could initialize `self._prerelease` to an empty string just to avoid exceptions below.